Update Github Issue Template: Bug (#8412)
RPCS3/rpcs3: .github/ISSUE_TEMPLATE/2-bug-report.md

---

<!---

> Please do not ask for help or general inquiry here, use RPCS3 Discord server instead.

Reviewer: This should probably be preserved somehow, to avoid support requests on tracker

Author: It was commented out for some reason. So I decided to remove it completely with redesign.
Maybe it's worth preserving as 0 point in the list?

Reviewer: It was commented out in order to show on the beginning of the issue template as soon as you create a new issue but not be visible if you posted the issue without removing the comment

Reviewer: Unsure what the best way to add it is, probably adding as point 0 suffices

Author: Ok, will try it now.

## Details

Please describe the problem as accurately as possible.

> #### 1. Please attach RPCS3 log

Reviewer 2: ```suggestion
#### 1. Please attach RPCS3's log.
```

> * Run the game

Reviewer 2: ```suggestion
* Run the game until you find the issue.
```

* Completely close RPCS3, or move to the next step in case it has crashed.

> * Locate RPCS3 log file

Reviewer 2: ```suggestion
* Locate RPCS3's log file:
```

	+ ```RPCS3.log``` (It can show up as just RPCS3 and have a notepad icon)
	or
	+ ```RPCS3.log.gz``` (It can show up as RPCS3.log and have a zip or rar icon)
![image](https://user-images.githubusercontent.com/44116740/84433247-aa15fa80-ac36-11ea-913e-6fe25a1d040e.png)
	On Linux it will be in ```~/.cache/rpcs3/```.
	On Windows it will be near the executable.
* Attach the log
	+ Drag and drop your log file into the issue.
	+ Or upload it to the cloud, such as Dropbox, Mega etc. 

> #### 2. If you describe graphical issue, please provide RSX capture and a RenderDoc capture that demonstrate the issue.

Reviewer 2: ```suggestion
#### 2. If you describe graphical issue, please provide an RSX capture and a RenderDoc capture that demonstrate it.
```

* To create an RSX capture, use _Create_ _RSX_ _Capture_ under _Utilities_.
Captures will be stored in RPCS3 folder → captures.
	+ Compress your capture with 7z, Rar etc. 
	And	drag and drop it into the issue.
	+ Or upload it to the cloud, such as Dropbox, Mega etc.

> * To create RenderDoc capture, please refer to [RenderDoc documentation](https://renderdoc.org/docs/how/how_capture_frame.html).

Reviewer 2: ```suggestion
* To create a RenderDoc capture, please refer to [RenderDoc's documentation](https://renderdoc.org/docs/how/how_capture_frame.html).
```

Author: Everything seems fair. Will correct in a sec.
